Collapsible/index.jsx

```diff
@@ -48,6 +43,20 @@ export default class Collapsible extends Component {
   }
 }
 
+/**
+ * Overflow rule to enable content to overflow outside the collapsible
+ * once the animation is close to be complete (the last few pixels take a while
+ * to be expanded). '10' is a magic number 🎩
+ */
+const shouldOverflow = (collapsed, animatedHeight, actualHeight) => {
+  const notYetAlmostExpanded = actualHeight - animatedHeight > 10
```

Comment:
This is a tiny little comment, and I think either way is okay, but for me personally I think a name like `notYetAlmostExpanded` is one negation too many. To me, it would be more clear if you reversed the comparison and then did `(collapsed || !almostExpanded)`.